e2e/commands/formOperations.ts

```diff
@@ -1,5 +1,5 @@
 import { APIRequestContext } from "@playwright/test";
 
-export const deleteForm = async (api: APIRequestContext, uuid: string) => {
+export async function deleteForm(api: APIRequestContext, uuid: string) {
```

Comment:
@denniskigen Why can't we use arrow functions here?

Reply:
No particular reason (except personal preference, maybe IIRC). Happy to revert if the former is more readable.

Reply:
Does everything else look ok, @anjula-sack?

Reply:
yes